docs/docsite/rst/scenario_guides/vmware_scenarios/scenario_clone_template.rst

```diff
@@ -132,7 +132,7 @@ What to expect
                 "ds_215"
             ],
             "hw_esxi_host": "192.0.2.44",
-            "hw_eth0": {
+            "Network adapter 1": {
```

Comment:
This is not a valid variable name. See https://docs.ansible.com/ansible/devel/user_guide/playbooks_variables.html#creating-valid-variable-names for more details.

Reply:
Ah ok, so I'll change these to maybe all lowercase and underscores and try again. Thanks for the info!